app/src/main/java/lt/vilnius/tvarkau/MainActivity.kt

```diff
@@ -0,0 +1,37 @@
+package lt.vilnius.tvarkau
+
+import android.os.Bundle
+import kotlinx.android.synthetic.main.app_bar.*
+import lt.vilnius.tvarkau.dagger.component.ApplicationComponent
+import lt.vilnius.tvarkau.dagger.component.MainActivityComponent
+import lt.vilnius.tvarkau.navigation.BottomNavigationController
+import javax.inject.Inject
+
+
+/**
+ * An activity representing a main activity home screen
+ */
+class MainActivity : BaseActivity() {
+
+    @Inject
+    lateinit var bottomNavigationController: BottomNavigationController
+
+    override fun onInject(applicationComponent: ApplicationComponent) {
+        MainActivityComponent.init(applicationComponent, this).inject(this)
```

Comment:
Yes. It will be done when I have some free time to finish. There is still a lot of to dos.